nucleus/admin/util/src/main/java/com/sun/enterprise/admin/remote/reader/ProgressStatusDTOJsonProprietaryReader.java

```diff
@@ -67,8 +69,7 @@ public ProgressStatusDTO readFrom(final HttpURLConnection urlConnection) throws
 
     @Override
     public ProgressStatusDTO readFrom(final InputStream is, final String contentType) throws IOException {
-        JsonParser jp = factory.createJsonParser(is);
-        try {
+        try (JsonParser jp = factory.createJsonParser(is)) {
```

Comment:
Just as a comment, but when you see uncommon abbreviations like "jp" here, best to rename it to something more clear right away